particula/aerosol_dynamics/parcel.py

```diff
@@ -1,21 +1,20 @@
-"""class for creating a air parcel to put particles and gases in.
+"""class for creating an air parcel to put particles and gases into.
 """
 
 import numpy as np
-from particula import u
```

Comment:
Can you bring this back and do some unit checks as decorators?

Reply:
We can talk about how we want to handle units, based on @wkchuang pull draft. I ran into unit errors/mismatches when assigning units in the inputs (so I removed them). I think it came from reloading the module and having inconsistent instances of unitreg, but not completely sure.

Reply:
I added back the unit check for radius. The mismatch was due to multiple instances of unitreg during modifying parcel.py. 
We do not need 'from particula import u' in as either the units will be in the input or the particle class will assign them, no units get created in parcel class. We don't need 'u' to do the unit checks, with the current try-exempt check being used.

Reply:
Maybe we should consider taking all units out and place them in the test suite. I think that makes significantly more sense to me than having units around in the actual routines. We will keep talking about this in PR #84